dapps/digital-twin-detail/src/TwinDAppComponent.ts

```diff
@@ -0,0 +1,41 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+// vue imports
+import Component, { mixins } from 'vue-class-component';
+
+// evan.network imports
+import { EvanComponent } from '@evan.network/ui-vue-core';
+
+// intenral imports
+import DAppTwin from './DAppTwin';
+import DAppContainer from './DAppContainer';
+
+/**
+ * Used to handle correct typings for the twin detail dapp.
+ */
+@Component
+export default class TwinDAppComponent extends mixins(EvanComponent) {
```

Comment:
I would avoid another inheritance hierarchy if possible. Instead we should rework our vuex store (or wait until it works better in vue3). If the store is available in all EvanComponents. then we could just put it there?

Reply:
Would not prefer to move this to the global store. In this case, it's more a specific store for this DApp. I can try to remove the inheritance by adjusting the vue initialization, to use a different vuex store interface for this dapp.